econet_heatpump_water_heater.yaml

```diff
@@ -127,6 +127,14 @@ sensor:
     device_class: "temperature"
     state_class: "measurement"
     entity_category: "diagnostic"
+  - platform: econet
+    name: "Active Alerts"
+    sensor_datapoint: ALRMALRT
+    accuracy_decimals: 0
+    icon: "mdi:water-boiler-alert"
+    state_class: "measurement"
+    entity_category: "diagnostic"
+    disabled_by_default: "true"
```

Comment:
I don't think this is safe without testing. My understanding is ESPHome will still create the component for this and try to update it. It will simply not be added to HA UI. Not sure if things will break, and more importantly how, since we don't know the type of this data. You added this as a sensor which works for FLOAT and ENUM_TEXT datapoint types. What if it's TEXT? Since we don't have another TEXT datapoint type there is some commented out code.

It would be safer to go with the C++ change and on your own config add this sensor until we verify this works. You could even add it both as a sensor and as a text_sensor. Or even a binary_sensor.

Reply:
Alright, this seems fine. I would like to get the C++ change in as it make sit much easier if I can use the public component rather than having my own fork locally.